src/pubsub/ua_pubsub_readergroup.c

```diff
@@ -656,8 +656,11 @@ UA_ReaderGroup_subscribeCallback(UA_Server *server,
         UA_ReaderGroup_setPubSubState(server, UA_PUBSUBSTATE_ERROR, readerGroup);
         return;
     }
-
+#ifdef UA_ENABLE_JSON_ENCODING
+    receiveBufferedNetworkMessageJson(server, readerGroup, connection);
```

Comment:
If `UA_ENABLE_JSON_ENCODING` is enabled, we still need to support the binary encoding.
So add an if/else here that looks at the configured encoding.